BUILD

```diff
@@ -2352,6 +2352,160 @@ grpc_cc_library(
     ],
 )
 
+grpc_cc_library(
+    name = "iomgr_ee_poller_common_hdrs",
```

Comment:
I'd probably just call this 'iomgr_ee_event_poller' - i think the _hdrs targets are an anti-pattern we should be avoiding - these are targets that export an interface, and that they don't have cc files is a coincidence rather than something that should be part of their name

Reply:
Done.